service/src/main/kotlin/fi/espoo/evaka/invoicing/controller/VoucherValueDecisionController.kt

```diff
@@ -108,6 +109,18 @@ class VoucherValueDecisionController(
         return ResponseEntity.ok(Wrapper(res))
     }
 
+    @GetMapping("/head-of-family/{headOfFamilyId}")
+    fun getHeadOfFamilyDecisions(
+        db: Database.Connection,
+        user: AuthenticatedUser,
+        @PathVariable headOfFamilyId: UUID
+    ): ResponseEntity<List<VoucherValueDecisionSummary>> {
+        Audit.VoucherValueDecisionHeadOfFamilyRead.log(targetId = headOfFamilyId)
+        user.requireOneOfRoles(UserRole.FINANCE_ADMIN)
```

Comment:
could include ADMIN too?

Reply:
I don't know if we should write new code with the action model or is it easier to refactor one full feature at a time. Probably the latter.

Reply:
Yeah, it should include `ADMIN`.